src/mpid/ch4/netmod/ofi/ofi_progress.h

```diff
@@ -88,7 +88,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_progress(int vci, int blocking)
 
     if (unlikely(MPIDI_OFI_has_cq_buffered(vni))) {
         ret = MPIDI_OFI_get_buffered(vni, wc);
```

Comment:
Rename or use a new variable named as `num` or something similar?

Reply:
Sure. Since this is already how `ret` is used in the non-buffered case, I'll just make it a separate commit.